### PR TITLE
Close un-used session

### PR DESCRIPTION
**Pull request by suqinhuang**
"Denying new connections due to the limit on number of connections" error
if don't close the session

Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
id: 1231094 1231096

**Comment by suqinhuang** [comment]
PASS 1-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viorng.during_bg_test.uninstall_install

PASS 2-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viorng.during_bg_test.downgrade_upgra

**Comment by suqinhuang** [comment]
@pingl2016 could you help review this patch

File: qemu/tests/single_driver_install.py [hunk 1]
```diff
@@ -140,19 +139,20 @@ def install_driver(session, operation):
...
+        session.close()
+        session = reboot(vm)
 
         uninstall_flag = params.get("need_uninstall", "no")
         if uninstall_flag == "yes":
             operation = "uninstall_driver"
             error_context.context("Uninstall driver", logging.info)
             install_driver(session, operation)
-            session = reboot(vm, session)
+            session = reboot(vm)
```
Comment (pingl2016) [inline qemu/tests/single_driver_install.py]: shall we need to close session before reboot()?

Comment (suqinhuang) [inline qemu/tests/single_driver_install.py]: the existing session in guest is closed after reboot

**Comment by pingl2016** [comment]
Acked-by: Ping Li \<pingl@redhat.com\>
(1/5) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.single_driver_install.with_netkvm:PASS (209.25 s)
 (2/5) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.single_driver_install.with_viorng:PASS (200.66 s)
 (3/5) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.single_driver_install.with_vioscsi: PASS (254.88 s)
 (4/5) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.single_driver_install.with_vioserial: PASS (211.90 s)
 (5/5) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.single_driver_install.with_balloon: PASS (184.05 s)

(02/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.before_bg_test.uninstall_install:  PASS (928.97 s)
 (03/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.before_bg_test.downgrade_upgrade:  PASS (910.83 s)
 (04/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.during_bg_test.uninstall_install:  PASS (492.40 s)
 (05/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.during_bg_test.downgrade_upgrade:  PASS (510.88 s)
 (06/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.after_bg_test.uninstall_install:  PASS (880.22 s)
 (07/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.after_bg_test.downgrade_upgrade:  PASS (886.58 s)
 (08/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.before_bg_test.uninstall_install:  PASS (758.20 s)
 (09/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.before_bg_test.downgrade_upgrade:  PASS (776.68 s)
 (10/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.during_bg_test.uninstall_install:  PASS (480.94 s)
 (11/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.during_bg_test.downgrade_upgrade:  PASS (506.17 s)
 (12/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.after_bg_test.uninstall_install:  PASS (710.04 s)
 (13/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_netkvm.after_bg_test.downgrade_upgrade:  PASS (727.45 s)
 (14/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.before_bg_test.uninstall_install:  PASS (733.51 s)
 (15/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.before_bg_test.downgrade_upgrade:  PASS (748.47 s)
 (16/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.uninstall_install:  PASS (572.60 s)
 (17/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.downgrade_upgrade:  PASS (625.10 s)
 (18/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.after_bg_test.uninstall_install:  PASS (653.87 s)
 (19/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.after_bg_test.downgrade_upgrade:  PASS (660.23 s)
 (20/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.before_bg_test.uninstall_install:  PASS (721.04 s)
 (21/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.before_bg_test.downgrade_upgrade:  PASS (762.56 s)
 (22/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.during_bg_test.uninstall_install:  PASS (559.00 s)
 (23/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.during_bg_test.downgrade_upgrade:  PASS (593.22 s)
 (24/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.after_bg_test.uninstall_install:  PASS (737.53 s)
 (25/25) smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioscsi.after_bg_test.downgrade_upgrade:  PASS (706.09 s)

**Pull request merged**